### PR TITLE
Feat(Core|Data): Show capture icon for capturable monsters only (Rise)

### PR DESCRIPTION
**Pull request by WetU**

### CODE REVIEW

File: HunterPie.Core/Game/Rise/Entities/MHRMonster.cs [hunk 1]
```diff
@@ -268,13 +268,26 @@ private void GetMonsterHealthData()
...
     private void GetMonsterCaptureThreshold()
     {
+        if (MonsterType == MonsterType.Qurio)
+        {
+            CaptureThreshold = 0.0f;
+            return;
+        }
+
+        MonsterDataSchema? data = MonsterData.GetMonsterData(Id);
+        if (data.HasValue && data.Value.Capture == 0)
```
Comment (Haato3o) [inline HunterPie.Core/Game/Rise/Entities/MHRMonster.cs:278]: I think a better approach would be to have a new field in the xml, like `IsCapturable` or something along these lines. It's better than using `Capture` as a boolean.

Comment (WetU) [reply HunterPie.Core/Game/Rise/Entities/MHRMonster.cs:278]: I was trying to not add or change something more. If I were to add an IsNotCapturable field, would it be a Bool value?

Comment (Haato3o) [reply HunterPie.Core/Game/Rise/Entities/MHRMonster.cs:278]: Yes, either a `IsNotCapturable` with default value as `false` or a `IsCapturable` with default value as `true` in the parser.

Comment (WetU) [reply HunterPie.Core/Game/Rise/Entities/MHRMonster.cs]: Sorry for taking time to work. Please review.

**Review by Haato3o — APPROVED** [review]

**Pull request merged**